doc: add known issues (#13)
nrfconnect/ncs-zigbee: docs/known_issues.rst

.. _known_issues:

Known issues
############

.. contents::
   :local:
   :depth: 3

Known issues listed on this page *and* tagged with the :ref:`latest release version <release_notes>` are valid for the current state of development.
Use the drop-down filter to see known issues for previous releases and check if they are still valid.

A known issue can list one or both of the following entries:

* **Affected platforms:**

  If a known issue does not have any specific platforms listed, it is valid for all hardware platforms.

* **Workaround:**

  Some known issues have a workaround.
  Sometimes, they are discovered later and added over time.

.. version-filter::
  :default: v1-2-1
  :container: dl/dt
  :tags: [("wontfix", "Won't fix")]

.. page-filter::
  :name: issues

  wontfix    Won't fix

List of known issues
********************

.. rst-class:: v1-2-1 v1-2-0 v1-1-0 v1-0-0

KRKNWK-20726: Too frequent keep-alive messages from SED to parent
  When using Light Switch sample application with sleepy end device (SED) configuration, the SED sends End Device Timeout Requests to its parent too frequently, when the parent supports the End Device Timeout Request Keepalive method.

>   This leads to increased power consumption on the SED compared to the case in which the parent only supports the MAC Data Poll Keepalive method, given the lenght of the requests and the need for reception of End Device Timeout Responses.

Reviewer: ```suggestion
  This increases SED power consumption compared to using only the MAC Data Poll Keepalive method, because requests are longer and End Device Timeout Responses must be received.
```
